packages/script-editor/package.json

```diff
@@ -50,7 +52,13 @@
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
     "rimraf": "^3.0.2",
-    "typescript": "~4.1.3"
+    "typescript": "~4.1.3",
+    "@jupyterlab/testutils": "3.3.0",
+    "@types/jest": "^23.3.11",
+    "install": "^0.13.0",
```

Comment:
Was this dependency on purpose?

Reply:
Yeah I added all of those suggested in the dev docs for setting up jest on a package https://github.com/elyra-ai/elyra/blob/master/docs/source/developer_guide/contributing.md#ui-tests

Since I noticed we are not using anything related to `enzyme` and removed related dependencies from that doc in this PR, do you think there's anything else we don't need?

Reply:
Oh huh! I don't see us importing it anywhere so we can probably remove it (unless it breaks without it) - same for anything else we're adding from that doc.

Reply:
This kind of relates to #681 - we should probably go over some of the dependencies in general and make sure we're not installing anything we aren't actually using (that's out of the scope of this PR of course 😄 )

Reply:
Ok I'll double check the other ones we are listing in that doc just in case

Reply:
Removed the dependency and all unit tests still pass locally, waiting for CI. The other dependencies listed seem to be in use.